### ISSUE TITLE
Status 500, asset shows parent file, asset shows in dassco UI

### ISSUE THREAD

**Issue by Baeist** [NHMDenmark/DaSSCo-asset-service#103]
The derivative with asset guid: 7e7-a-02-0e-1d-0d-1-001-01-000-015753-00000_72 institution: AU Collection: Vascular plants

When calling endpoints get asset, get asset status i get a status 500 back. 

When searching the asset in the UI the asset gets found but then when you try to open it the asset is not found message is dsiplayed. 

When asking for the available file list of the asset the parent asset shows up. /assetfiles/AU/Vascular plants/7e7-a-02-0e-1d-0d-1-001-01-000-015753-00000_72

I have logs showing this asset receiving a 504 during creation and 400 when attempting a sync with erda. The 504 was checked or retried and seemingly worked. Doing a manual sync also gives the 400 currently with the asset response saying "no writeable share for asset found". I can see that the file upload failed with the same status.

**Comment by Grand666** [comment]
Hi @Baeist 

When you get a 504, do you then have some sort of retry logic?

**Comment by Baeist** [comment]
Yes, first there is a check if the asset was created anyway. Then if not the asset is rescheduled to be created again.

**Comment by Baeist** [comment]
When looking at the get in progress assets this asset shows up twice. So its probably a duplicate in the database scenario.

**Comment by Grand666** [comment]
Hi Lars,

The fix was deployed after this asset was created. (When you created this task)

I just forgot to remove the duplicate of this "7e7-a-02-0e-1d-0d-1-001-01-000-015753-00000_72". I will do that tomorrow, when we'll deploy bug fix for the Graph/Statistics.

Have you seen any new instances of duplicates?
